<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <item row="0" column="1" colspan="4">
    <widget class="QStackedWidget" name="pageWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>1</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="pageProfile">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Tracker Account</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="loginLabel">
              <property name="text">
               <string>Login:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="loginLineEdit"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordLineEdit">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>216</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="registerLink">
            <property name="text">
             <string>&lt;a style='text-decoration:none; color:#1a4d82;' href=&quot;#&quot;&gt;Register&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBoxKP">
         <property name="title">
          <string>KINOPOISK.RU Account</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="loginLabel_2">
              <property name="text">
               <string>Login:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="loginLineEditKP"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel_2">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordLineEditKP">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>178</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="registerLinkKP">
            <property name="text">
             <string>&lt;a style='text-decoration:none; color:#1a4d82;' href=&quot;http://www.kinopoisk.ru/level/30/#new&quot;&gt;Register&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBoxIMDB">
         <property name="title">
          <string>Tracker Account</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0" colspan="2">
           <layout class="QFormLayout" name="formLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="loginLabel_3">
              <property name="text">
               <string>Email:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="loginLineEditIMDB"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="passwordLabel_3">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="passwordLineEditIMDB">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>178</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="registerLinkIMDB">
            <property name="text">
             <string>&lt;a style='text-decoration:none; color:#1a4d82;' href=&quot;https://secure.imdb.com/register-imdb/?why=personalize&quot;&gt;Register&lt;/a&gt;</string>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageOther">
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Select BitTorrent client</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="clientLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="selectClientButton">
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2" colspan="5">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>232</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QPushButton" name="resetButton">
     <property name="maximumSize">
      <size>
       <width>31</width>
       <height>25</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Reset all settings</string>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../data/uploader.qrc">
       <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2">
    <widget class="QPushButton" name="okButton">
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>OK</string>
     </property>
     <property name="icon">
      <iconset resource="../../../data/uploader.qrc">
       <normaloff>:/images/dialog-ok.png</normaloff>:/images/dialog-ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="4" rowspan="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
     <property name="icon">
      <iconset resource="../../../data/uploader.qrc">
       <normaloff>:/images/dialog-cancel.png</normaloff>:/images/dialog-cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QListWidget" name="menuList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>142</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>142</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="iconSize">
      <size>
       <width>16</width>
       <height>16</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>Profiles</string>
      </property>
      <property name="icon">
       <iconset resource="../../../data/uploader.qrc">
        <normaloff>:/images/profile.png</normaloff>:/images/profile.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
      <property name="icon">
       <iconset resource="../../../data/uploader.qrc">
        <normaloff>:/images/application-x-desktop.png</normaloff>:/images/application-x-desktop.png</iconset>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>menuList</tabstop>
  <tabstop>loginLineEdit</tabstop>
  <tabstop>passwordLineEdit</tabstop>
  <tabstop>groupBoxKP</tabstop>
  <tabstop>loginLineEditKP</tabstop>
  <tabstop>passwordLineEditKP</tabstop>
  <tabstop>groupBoxIMDB</tabstop>
  <tabstop>loginLineEditIMDB</tabstop>
  <tabstop>passwordLineEditIMDB</tabstop>
  <tabstop>resetButton</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>clientLineEdit</tabstop>
  <tabstop>selectClientButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../data/uploader.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>400</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>410</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
